README.md

```diff
@@ -3,154 +3,110 @@ This repository holds the kcp tests that tests against the publicly available  i
+* Golang installed. See [Installing Golang](https://golang.org/doc/install), the newer the better ( Ensure you install Golang from a binary release [found here](https://golang.org/dl/), not with a package manager such as `dnf` ）
 * golint installed. See [Installing golint](https://github.com/golang/lint#installation)
-* Have the environment variable `KUBECONFIG` set pointing to your cluster
-
-### Include new test folder
-If you create a new folder for your test case, **add the path** to the [include.go](https://github.com/openshift/openshift-tests-private/blob/master/test/extended/include.go)
-
-### Create go-bindata for new YAML files
-If you have some **new YAML files** used in your code, you have to generate the bindata first.
-Run `make update` to update the bindata. For example, you can see the bindata has been updated after running the `make update` as follows:
-```console
-$ git status
-	modified:   test/extended/testdata/bindata.go
-	new file:   test/extended/testdata/olm/etcd-subscription-manual.yaml
-```
-
-### Compile the executable binary
-Note that we use the `go module` for package management, the previous `go path` is deprecated.
-```console
-$ git clone git@github.com:openshift/openshift-tests-private.git
-$ cd openshift-tests-private/
-$ make build
-mkdir -p "bin"
-export GO111MODULE="on" && export GOFLAGS="" && go build -o "bin" "./cmd/extended-platform-tests"
-$ ls -hl ./bin/extended-platform-tests 
--rwxrwxr-x. 1 cloud-user cloud-user 165M Jun 24 22:17 ./bin/extended-platform-tests
-```
-
+* Install [kubelogin](https://github.com/int128/kubelogin.git) and [KCP kubectl plugin](https://github.com/kcp-dev/kcp.git) ( Ensure your installed KCP kubectl plugin version is the same with the kcp service version )
+  - If you would like to download the suitable version packages directly go to [kcp release assets](https://github.com/kcp-dev/kcp/releases), download the zip file suitable for your OS, extract and copy the binaries to a place which is in your PATH and once copied run `kubectl-kcp --version` to verify if server matches with client 
+  - If you would like to build the plugin by your self you could follow the steps like below:
+  ```
+  $ git clone https://github.com/kcp-dev/kcp.git
+  # E.g. Install the release-0.5 version KCP kubectl plugin
+  # commitID: "fccfb61" of release-0.5 branch/tag
+  $ git checkout fccfb61
+  $ make install WHAT=./cmd/kubectl-kcp && make install WHAT=./cmd/kubectl-workspaces && make install WHAT=./cmd/kubectl-ws
+  ```
+
+* Have the environment variable `KUBECONFIG` set pointing to your kcp service
+* Login the kcp service via SSO (Single Sign On)
+  ```
+  kubectl oidc-login get-token --oidc-issuer-url=https://sso.xxxx.com/auth
+   --oidc-client-id=cloud-services --oidc-redirect-url-hostname=127.0.0.1
+  ```
+* If you would like to test BYO cases please make sure to set the `PCLUSTER_KUBECONFIG` environment variable else all the tests related to BYO will be skipped. 
```

Comment:
```suggestion
* If you would like to test BYO cases please make sure the `PCLUSTER_KUBECONFIG` environment variable is set otherwise all the tests related to BYO will be skipped. 
```

Reply:
Addressed.